src/Ticket.php

```diff
@@ -446,7 +446,7 @@ public function getDatasToAddSLA($slas_id, $entities_id, $date, $type)
             }
            // Compute time_to_resolve
             $data['sla_waiting_duration'] = $this->fields['sla_waiting_duration'] ?? 0;
-            $data[$dateField]             = $sla->computeDate($date, $data['sla_waiting_duration']);
+            $data[$dateField]             = $sla->computeDate($date, (int) $data['sla_waiting_duration']);
```

Comment:
`computeDate` is apparently called from elsewhere; it may be better to force typehinting in the method itself (or you can also fix other core calls).
Note I have no idea if that can be called form plugins (I only have a few ones locally).

Reply:
We could indeed cast the value in `computeDate` method itself.